blazor/gantt-chart/row-selection.md

```diff
@@ -221,6 +221,51 @@ You can select multiple rows by setting the `SelectionSettings.Type` property to
 
 ![Multiple Row Selection in Blazor Gantt Chart](images/blazor-gantt-chart-multi-row-selection.png)
 
+## Hover row selection
```

Comment:
CHANGE heading to "Hover effect on rows"

Reply:
Updated